tests/expectations/tests/macro-redef.rs

```diff
@@ -0,0 +1,9 @@
+/* automatically generated by rust-bindgen */
+
+
+#![allow(non_snake_case)]
+
+
+pub const FOO: ::std::os::raw::c_uint = 4;
```

Comment:
Ideally, `FOO` would be `5` in the bindings. We want the last definition, not the first.

Also, file a follow up issue to track not generating bindings for macros that have been `#undef`ed.